packages/protocol/contracts/governance/Election.sol

```diff
@@ -684,13 +751,24 @@ contract Election is
    * @param greater The group receiving more votes than the group for which the vote was revoked,
    *   or 0 if that group has the most votes of any validator group.
    */
-  function decrementTotalVotes(address group, uint256 value, address lesser, address greater)
-    private
-  {
+  function decrementTotalVotes(
+    address account,
+    address group,
+    uint256 value,
+    address lesser,
+    address greater
+  ) private {
     if (votes.total.eligible.contains(group)) {
       uint256 newVoteTotal = votes.total.eligible.getValue(group).sub(value);
       votes.total.eligible.update(group, newVoteTotal, lesser, greater);
     }
+
+    CachedVotes storage cachedVotes = cachedVotesByAccount[account];
```

Comment:
This doesn't need to happen for groups that vote using the max groups limitation. Might be worth skipping this for accounts with default strategy, as there are some storage reads/writes here.